### ISSUE TITLE
META should refer to 3.1, not 3.0

### ISSUE THREAD

**Issue by davidpichardie** [javalib-team/javalib#10]
https://github.com/javalib-team/javalib/blob/bfd68740c7f8e66fa44f93de6f753f6fc8c1d8e4/src/META.source#L1

**Comment by dmitris** [comment]
Is this the reason for the install failure (on Mac):
```
$ opam install javalib
The following actions will be performed:
  ∗ install javalib 3.1

<><> Gathering sources ><><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫
[javalib.3.1] found in cache

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫
[ERROR] The installation of javalib failed at "make install".

#=== ERROR while installing javalib.3.1 =======================================#
# context     2.0.5 | macos/x86_64 | ocaml-base-compiler.4.08.1 | https://opam.ocaml.org#b70af589
# path        ~/.opam/4.08.1/.opam-switch/build/javalib.3.1
# command     ~/.opam/opam-init/hooks/sandbox.sh install make install
# exit-code   2
# env-file    ~/.opam/log/javalib-85369-6d07ae.env
# output-file ~/.opam/log/javalib-85369-6d07ae.out
### output ###
# make -C src install
# /Users/dmitris/.opam/4.08.1/bin/ocamlfind install javalib META javalib.cma javalib.cmi javalib.cmxa javalib_pack.cmi javalib_pack.cmo javalib_pack.cmx javalib_pack.o ptrees/ptset.mli ptrees/ptmap.mli ptrees/genericSet.mli ptrees/genericMap.mli jLib.mli jBasics.mli jBasicsLow.mli jSignature.mli jCode.mli jClassLow.mli jClass.mli jDumpBasics.mli jUnparseSignature.mli jDumpLow.mli jParseCode.ml[...]
# ocamlfind: Package javalib is already installed
#  - (file /Users/dmitris/.opam/4.08.1/lib/javalib/META already exists)
# make[1]: *** [install] Error 2
# make: *** [install] Error 2



<><> Error report <><><><><><><><><><><><><><><><><><><><><><><><><><><><><>  🐫
┌─ The following actions failed
│ ∗ install javalib 3.1
└─
╶─ No changes have been performed
```

**Comment by anscoil** [comment]
I am able to reproduce this bug on linux only by doing `make install` manually in the javalib repository first. Then `opam install javalib` will fail as in your example.
But there is clearly a mistake in the META file which is going to be fixed in javalib 3.1.1.

**Comment by dmitris** [comment]
I see there's a commit https://github.com/javalib-team/javalib/commit/8e55c29e2975089409276d895cf8fdec6489a86b - hopefully 3.1.1 is out soon 😄
